src/backend/utils/systeminfo/steamDeck.ts

```diff
@@ -7,7 +7,11 @@ type SteamDeckInfo =
   | { isDeck: false; model?: undefined }
 
 function getSteamDeckInfo(cpus: CpuInfo[], gpus: GPUInfo[]): SteamDeckInfo {
-  if (cpus[0]?.model !== 'AMD Custom APU 0405') return { isDeck: false }
+  if (
+    cpus[0]?.model !== 'AMD Custom APU 0405' &&
+    cpus[0]?.model !== 'AMD Custom APU 0932'
+  )
+    return { isDeck: false }
```

Comment:
I'm checking on my steamdeck and it looks like we have a `HOSTNAME` env variable with the value `steamdeck`, maybe we can just use that info for this instead? I imagine it's more future proof

what do you think @CommandMC ?

We have other variables like:
```
HOME=/home/deck
LOGNAME=deck
```

and many env variables including `/home/deck` or `steamdeck`

Reply:
`HOSTNAME` is changable from the steam ui itself and I haven't checked how multi users are handled on the deck (may have different `HOME`?). 

Problem I see with the environment variables is that anyone running steamos image on their machine would have the same set, and hardware ID is harder to spoof. (If someone did it, then he had his purpose)

Reply:
> I haven't checked how multi users are handled on the deck

As far as I can tell, they aren't, you're always the `deck` user account

Regardless, I agree, we should only look at things definitively proving we're on the Steam Deck hardware, not just SteamOS or something that's very similar to what the Deck might look like